### PR TITLE
Update simulate.sh to bypass auth

### PR DESCRIPTION
**Pull request by dluc**
# Change type 

- [x] Bug fix
- [ ] New feature
- [ ] Enhancement
- [ ] Breaking change (breaks backward compatibility)

**Checklist:**

- [ ] All tests passed
- [ ] The code follows the code style and conventions of this project
- [ ] The change requires a change to the documentation
- [ ] I have updated the documentation accordingly

# Description and Motivation 
Fix for https://github.com/Azure/device-simulation-dotnet/issues/66 , simulate.sh now works

---
This change is [Reviewable](https://reviewable.io/reviews/azure/pcs-cli/174)

### CODE REVIEW

File: remotemonitoring/single-vm/scripts/simulate.sh [hunk 1]
```diff
@@ -1,19 +1,13 @@
...
 
-echo "Script temporarily disabled, see https://github.com/Azure/device-simulation-dotnet/issues/66"
-exit -1
-
 echo "Starting simulation..."
-ISUP=$(curl -sk https://localhost/devicesimulation/v1/status | grep "Alive" | wc -l)
+ISUP=$(docker exec -it app_devicesimulation_1 curl -sk http://localhost:9003/v1/status | grep "Alive" | wc -l)
 while [[ "$ISUP" == "0" ]]; do
   echo "Waiting for simulation service to be available..."
   sleep 4
```
Comment (timlaverty) [inline remotemonitoring/single-vm/scripts/simulate.sh]: 4s or 4 ms?

File: remotemonitoring/single-vm/scripts/simulate.sh [hunk 2]
```diff
@@ -1,19 +1,13 @@
...
-  ISUP=$(curl -sk https://localhost/devicesimulation/v1/status | grep "Alive" | wc -l)
+  ISUP=$(docker exec -it app_devicesimulation_1 curl -sk http://localhost:9003/v1/status |grep "Alive" | wc -l)
```
Comment (timlaverty) [inline remotemonitoring/single-vm/scripts/simulate.sh]: this is a bit brittle; i.e. it's dependent on the response string Alive being in there.  Probably fine but is there something else we should key off of?  e.g. should we just use a 200/OK?

File: remotemonitoring/single-vm/scripts/simulate.sh (hunk 1 above)
Comment (dluc) [reply remotemonitoring/single-vm/scripts/simulate.sh]: 4 seconds

File: remotemonitoring/single-vm/scripts/simulate.sh (hunk 2 above)
Comment (dluc) [reply remotemonitoring/single-vm/scripts/simulate.sh]: sure, there are things we can improve, status code should do but I don't want to change that now. I'll track it as a separate task

**Pull request merged**